consensus-types/blocks/proto_test.go

```diff
@@ -934,6 +1198,48 @@ func bodyPbBlindedCapella() *eth.BlindedBeaconBlockBodyCapella {
 	}
 }
 
+func bodyPbDeneb() *eth.BeaconBlockBodyDeneb {
+	f := getFields()
+	return &eth.BeaconBlockBodyDeneb{
+		RandaoReveal: f.sig[:],
+		Eth1Data: &eth.Eth1Data{
+			DepositRoot:  f.root[:],
+			DepositCount: 128,
+			BlockHash:    f.root[:],
+		},
+		Graffiti:              f.root[:],
+		ProposerSlashings:     f.proposerSlashings,
+		AttesterSlashings:     f.attesterSlashings,
+		Attestations:          f.atts,
+		Deposits:              f.deposits,
+		VoluntaryExits:        f.voluntaryExits,
+		SyncAggregate:         f.syncAggregate,
+		ExecutionPayload:      f.execPayloadDeneb,
+		BlsToExecutionChanges: f.blsToExecutionChanges,
```

Comment:
You should create some KZG commitments inside `getFields()` and populate them here